backends/xnnpack/serialization/runtime_schema.fbs

```diff
@@ -63,7 +65,9 @@ table PerChannelGroupQuant {
   scale:[float];
   channel_dim:int;
   group_size:int;
-  scale_bf16:[ushort];
+  scale_bf16:[ushort] (deprecated);
```

Comment:
curious, why mark this as deprecated but not float if we are moving to ndm for evreythig?

Reply:
because this actually was never used, since we never added the export path to serialize into this field. scale[float] is still used by older versions